### ISSUE TITLE
Create toggle choice between /collections displaying all collections or just top-level collections

### ISSUE THREAD

**Issue by AmandaRust** [NEU-Libraries/drs-toolkit-wp-plugin#217]
Suggestion for HAC and Our Marathon projects:
The /collections path currently brings up all collections, including nested collections. We might want to consider having this page display the top-level collections only rather than all collections, including nested collections.

For example, for Arader: https://arader.library.northeastern.edu/collections

Rather than pulling up the list of all collections in the base URL: 

![2018-03-20_1448](https://user-images.githubusercontent.com/4663073/37676250-cab4d254-2c4d-11e8-9997-5feaa5fcdae0.png)

Just the top-level collections should display:
![2018-03-20_1448-1](https://user-images.githubusercontent.com/4663073/37676274-d80901e6-2c4d-11e8-9147-eea9494688ce.png)

Or, toggling between all collections and just top level collections could be a CERES setting.

_Originally posted by @sarahjeansweeney in https://github.com/NEU-DSG/toolkit_requests/issues/90#issuecomment-374714753_

**Comment by AmandaRust** [comment]
From a meeting today (August 12, 2019): additional example from Holocaust Awareness Committee collections page. They have a deeply nested collection, basically: material format --> lecture series --> year. Since the collections page displays all collections in a flat hierarchy, there are multiple year folders without clear context.

https://holocaustawarenessarchives.northeastern.edu/collections/

![image](https://user-images.githubusercontent.com/10200332/62884539-d30eeb80-bd04-11e9-8fce-08a045d425c7.png)

A browse experience only through the top-level collections would give the user more context; i.e., displaying just these collections and having user browse down:

![image](https://user-images.githubusercontent.com/10200332/62884596-f174e700-bd04-11e9-8521-1dfdb4d80a5a.png)

**Comment by AmandaRust** [comment]
^^^ By "these collections": the named collections like "Audio and Video", "Photopgraphs", etc.

**Comment by patrickmj** [comment]
I strongly agree with the original issue that this'd be best handled as a CERES setting, since each exhibit will have a different structure. 

I'm pretty sure that it would require some complex digging through the API response to unflatten the list of collections. It might even be a question for David and the DRS 2.0 API.
